dapp/.github/workflows/playwright.yml

```
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Install dependencies
+        run: npm ci
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+      - name: Run Playwright tests
+        run: npx playwright test
```

Please use Bun as everywhere and we might want to add some command in the package.json. This should be it:
```suggestion
      - uses: oven-sh/setup-bun@v2
      - name: Install App dependencies
        run: cd dapp && bun install
      - name: Install Playwright Browsers
        run: bun run playwright install --with-deps
      - name: Run Playwright tests
        run: bun run playwright test
```